certified-connectors/GetAccept/apiProperties.json

```diff
@@ -0,0 +1,59 @@
+{
+  "properties": {
+    "capabilities": [
+      "actions",
+      "triggers"
+    ],
+    "connectionParameters": {
+      "token": {
+        "oAuthSettings": {
+          "clientId": "Flow",
+          "customParameters": {
+            "authorizationUrlTemplate": {
+              "value": "https://app.getaccept.com/oauth2/authorize"
+            },
+            "tokenUrlTemplate": {
+              "value": "https://app.getaccept.com/oauth2/token"
+            },
+            "refreshUrlTemplate": {
+              "value": "https://app.getaccept.com/oauth2/token"
+            },
+            "refreshBodyTemplate": {
+              "value": "client_id={ClientId}&scope={Scopes}&refresh_token={RefreshToken}&redirect_uri={RedirectUrl}&grant_type=refresh_token&client_secret={ClientSecret}"
+            },
+            "scopeListDelimiter": {
+              "value": " "
+            }
+          },
+          "identityProvider": "oauth2generic",
+          "properties": {
+            "IsFirstParty": false
+          },
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "scopes": [
+            "basic"
+          ]
+        },
+        "type": "oauthSetting"
+      }
+    },
+    "iconBrandColor": "#393A42",
+    "publisher": "GetAccept, Inc.",
```

Comment:
@blanck  Please add `stackOwner` property

Reply:
@blanck Checking in on the above